### PR TITLE
fix(faq): correct glacier mentions MTA-1974

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**

### CODE REVIEW

File: faq/coldstorage.mdx [hunk 1]
```diff
@@ -1,98 +1,121 @@
...
-## How much do I pay to archive objects to Glacier or to restore them to Standard class?
+Scaleway’s cold storage service Glacier is engineered on specific hardware:
 
-Nothing. Switching the object class from `STANDARD` to `GLACIER` is free.
-Restoring objects from `GLACIER` to `STANDARD` is also free of charge.
+- SMR disks that increase storage density and overall per-drive storage capacity
+- motherboards manufactured internally that can power on/off disks on-demand with a SATA bus tree matrix.
 
-## I have started the restoration of an object, why is it not available instantaneously?
+This particular hardware configuration allows us to offer competitive pricing for use-cases that do not need the performance of other Object storage classes. The counterpart is that data stored on a cold storage service such as Scaleway Glacier has [limitations]() in relation to data retrieval.
```
Comment (nedaehm) [inline faq/coldstorage.mdx:20]: There's a missing link here for the limitations.

File: faq/coldstorage.mdx [hunk 2]
```diff
@@ -1,98 +1,121 @@
...
+ 
+When a lifecycle rule is set for a very large amount of objects (several 100000s) simultaneously, the rule might take longer to be applied.
+ 
 </Message>
 
+ 
 ## How do I encrypt my data?
-
+ 
 For increased security, we recommend encrypt your data before sending it to Scaleway Glacier.
```
Comment (nedaehm) [inline faq/coldstorage.mdx:86]:
```suggestion
For increased security, we recommend you encrypt your data before sending it to Scaleway Glacier.
```

File: faq/coldstorage.mdx [hunk 3]
```diff
@@ -1,98 +1,121 @@
...
-In addition, Scaleway Glacier enables parallel writing to multiple disks simultaneously, providing very high read and write speeds. 
-Redundancy is achieved by advanced parity mechanisms (6+3) on a large number of media, whereas tape technology does not allow this and is limited to CRC.
-Finally, there is no risk of mechanical part failure (ex: robot) or wear and tear (tape drive).
 
 ## Can I change the storage class of an entire bucket from Standard to Glacier?
-
+ 
 Yes, a best practice is to create a transition lifecycle rule with an empty prefix.
 In this case, all files contained within the selected bucket will have their storage class altered automatically on the dates stipulated by you.
 However, due to S3 Protocol restrictions, a lifecycle rule cannot be created to modify the storage class from glacier to standard.
```
Comment (nedaehm) [inline faq/coldstorage.mdx:120]:
```suggestion
However, due to S3 Protocol restrictions, a lifecycle rule cannot be created to modify the storage class from Glacier to Standard.
```

File: faq/coldstorage.mdx [hunk 4]
```diff
@@ -1,98 +1,121 @@
...
-Yes, there is a free tier and you can benefit from 75 GB of storage in Glacier class for free with no time limit. In addition, you can also benefit from  75 GB of storage and 75 GB of outgoing and inter-regional transfer in Object Storage’s standard class.
+The cold storage “Glacier” class of Object Storage gets its name in opposition to “hot data”. Hot data is the type that is frequently accessed. Cold storage is accessed less frequently and does not require fast access like hot data. 
 
-## How much do I pay to archive objects to Glacier or to restore them to Standard class?
+Scaleway’s cold storage service Glacier is engineered on specific hardware:
 
-Nothing. Switching the object class from `STANDARD` to `GLACIER` is free.
-Restoring objects from `GLACIER` to `STANDARD` is also free of charge.
+- SMR disks that increase storage density and overall per-drive storage capacity
+- motherboards manufactured internally that can power on/off disks on-demand with a SATA bus tree matrix.
```
Comment (RoRoJ) [inline faq/coldstorage.mdx:18]:
```suggestion
- motherboards manufactured internally that can power disks on/off on-demand with a SATA bus tree matrix.
```

File: faq/coldstorage.mdx [hunk 5]
```diff
@@ -1,98 +1,121 @@
...
 
-## I have started the restoration of an object, why is it not available instantaneously?
+This particular hardware configuration allows us to offer competitive pricing for use-cases that do not need the performance of other Object storage classes. The counterpart is that data stored on a cold storage service such as Scaleway Glacier has [limitations]() in relation to data retrieval.
+
+## When should I use Glacier class instead of Standard class?
+ 
+Scaleway Glacier is engineered for use-cases that can be described as “deep archiving”: a durable Object Storage class designed for data archiving, legal documents of long-term backups that do not require immediate access.
```
Comment (RoRoJ) [inline faq/coldstorage.mdx:24]:
```suggestion
Scaleway Glacier is engineered for use-cases that can be described as “deep archiving”. It's a durable Object Storage class designed for data archiving, for example legal documents or long-term backups that do not require immediate access.
```

File: faq/coldstorage.mdx [hunk 6]
```diff
@@ -1,98 +1,121 @@
...
-## When should I use Glacier class instead of Standard class?
+  It can take anywhere from a few seconds to 6 hours to retrieve the first byte of a requested file. From that point, file retrieval is limited to xMo/sec. Unfortunately, we are unable to provide estimates of how long data retrieval will therefore take for each use case.
 
-It depends on how you will use Object Storage. If you consider that a 3-month-old object is rarely used but still has a value, you might want to send it to the Glacier class, for example.
+</Message>
 
+ 
+## Is there a free tier on Scaleway Glacier?
+ 
+Yes, there is a free tier and you can benefit from 75 GB of storage in Glacier class for free with no time limit. In addition, you can also benefit from  75 GB of storage and 75 GB of outgoing and inter-regional transfer in Object Storage’s standard class.
```
Comment (RoRoJ) [inline faq/coldstorage.mdx:42]:
```suggestion
Yes, there is a free tier and you can benefit from 75 GB of storage in Glacier class for free with no time limit. In addition, you can also benefit from 75 GB of storage and 75 GB of outgoing and inter-regional transfer in Object Storage’s standard class.
```

File: faq/coldstorage.mdx [hunk 7]
```diff
@@ -1,98 +1,121 @@
...
 For example, a user wants to configure a lifecycle rule that moves an object after two days to Glacier:
-
-- Lifecycle Rules are checked everyday at midnight.
-- The user uploaded a file on April 6th, at 2:00 pm UTC.
-- The rule is triggered on April 8th, at 11:59 pm UTC.
-- The application of the rule can take up to 24 hours
-⇒ The object will be transferred to Glacier no later than 11:59 pm UTC on April 9th.
-
+ 
+- lifecycle Rules are checked everyday at midnight.
```
Comment (RoRoJ) [inline faq/coldstorage.mdx:71]: is it right that lifecycle isn't capitalized and rules is?

File: faq/coldstorage.mdx [hunk 8]
```diff
@@ -1,98 +1,121 @@
...
-- The deletion of data after a certain period of time
-- The archiving of data that must legally be kept
-Lifecycle management features meet both of these requirements by allowing you to set a timeframe beyond which your data are archived or deleted.
 
-## What is the maximum data volume and object size supported by Scaleway Glacier?
+- the deletion of data after a certain period of time
+- the archiving of data that must legally be kept
 
-There is no maximum data size, Glacier accepts an unlimited number of objects up to 5 TB each. 
+Lifecycle management features meet both of these requirements by allowing you to set a timeframe beyond which your data are archived or deleted.
```
Comment (RoRoJ) [inline faq/coldstorage.mdx:97]:
```suggestion
Lifecycle management features meet both of these requirements by allowing you to set a timeframe beyond which your data is archived or deleted.
```

**Review by ofranc — APPROVED** [review]

File: faq/coldstorage.mdx (hunk 5 above)
Comment (RoRoJ) [reply faq/coldstorage.mdx:24]: meh reading this back I don't like my suggestion either

**Review by bene2k1 — APPROVED** [review]

**Pull request merged**